entry/src/main/ets/InputMethodExtensionAbility/model/KeyboardController.ets

```diff
@@ -287,7 +292,9 @@ export class KeyboardController {
       return;
     }
     // 1260x2720 on emulator
-    const { width, height } = display.getDefaultDisplaySync()
+    const defaultDisplay = display.getDefaultDisplaySync()
+    const width = defaultDisplay.width
+    const height = defaultDisplay.height
```

Comment:
It's really wierd that ArkTS doesn't support this. It's no more than a syntax sugar.